CONTRIBUTING.md

```diff
@@ -58,14 +55,13 @@ Also, remove the "wrench" icon (🔧) from the TOC and the file heading, if appl
 
 If your PR is addressing an existing issue, add the `Fixes #Issue_Number` keyword to the commit message or PR description, so the issue can be automatically closed when the PR is merged. For more information, see [Closing issues via commit messages](https://help.github.com/articles/closing-issues-via-commit-messages/).
 
-The .NET team will review your PR and let you know if the change looks good or if there are any other updates/changes necessary in order to approve it.
-
-**Step 6:** Make any necessary updates to your branch as discussed with the team. 
+The .NET team will review your PR and let you know if there are any other updates/changes necessary in order to approve it.
 
-The maintainers will merge your PR into the master branch once feedback has been applied and your change looks good. 
+**Step 6:** Rerspond to the PR feedback.
 
-On a certain cadence, we push all commits from master branch into the live branch and then you'll be able to see your contribution live at https://docs.microsoft.com/dotnet/. 
+The maintainers will merge your PR into the master branch once feedback has been applied and your PR is approved.
 
+We periodically push all commits from master branch into the live branch. You'll be able to see your contribution at https://docs.microsoft.com/dotnet/, typically within a few hours of the merge.
```

Comment:
which merge are you referring to in the last sentence? the merge to master or the merge to live? we don't always merge to live the same day, so I'm just trying to not create false expectations here.